server/schedule/opts.go

```diff
@@ -21,6 +21,7 @@ import (
 type Options interface {
 	GetLeaderScheduleLimit() uint64
 	GetRegionScheduleLimit() uint64
+	GetTolerantSizeRatio() float64
```

Comment:
here you use tolerant size, but your TOML file use tolerate size, please choose one.